src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -13,6 +13,31 @@
                     #{studyId}
                 </foreach>
             </if>
+            <!-- sample-lists-counts filter -->
+            <if test="studyViewFilter.caseLists != null and !studyViewFilter.caseLists.isEmpty()">
+                INTERSECT
+                <!-- get samples that are in each sample list in the filter -->
+                SELECT * FROM (
+                <foreach item="ANDGroup" collection="studyViewFilter.caseLists" separator="INTERSECT">
```

Comment:
this may also be true of my sample-list PR, have to check, but I think this SQL may be suboptimal because it selects from the big table (sample_list_list) multiple times (foreach).     i think it would be much faster to just do something like:
```
SELECT sampleId FROM sample_list_list WHERE caseLists in (caseList1, caseList2 ...) 
```
if you think about this in a coding perspective, you want to iterate through the big list once rather than iterating multiple times.   lets test this theory in SQL client first